### PR TITLE
Upgraded 0.0.3 -> 0.0.4

### PR DESCRIPTION
**Pull request by MurasakiArtemis**
Version bump

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='django-airbrake',
-    version='0.0.3',
+    version='0.0.4',
```
Comment (JoshPaulsen) [inline setup.py:6]: I have no idea if this is important but there is also a reference to version here: https://github.com/Captricity/airbrake-django/blob/master/django_airbrake/utils/client.py#L65

**Review by JoshPaulsen — COMMENTED** [review]
Update that other version (just in case) and I'll approve!

File: setup.py (hunk 1 above)
Comment (MurasakiArtemis) [reply setup.py:6]: Done

**Review by JoshPaulsen — APPROVED** [review]
Nice!

**Pull request merged**